Formula/pilosa.rb

```diff
@@ -15,13 +17,23 @@ class Pilosa < Formula
   depends_on "go" => :build
   depends_on "glide" => :build
 
+  go_resource "github.com/rakyll/statik" do
```

Comment:
Would it be preferable to create a homebrew package for statik (similar to the glide one) and depend on it?